src/test/daml/Daml/Finance/Settlement/Test/Transfer.daml

```diff
@@ -137,13 +144,16 @@ run2 = script do
   (batchCid2, [cashInstructionCid2]) <- submit bank do exerciseCmd settlementFactoryCid Factory.Instruct with instructors = singleton bank; settlers = singleton bank; id = id2; description = "transfer of 200000.0 USD payment"; contextId = None; steps = [second]
   (batchCid3, [cashInstructionCid3]) <- submit bank do exerciseCmd settlementFactoryCid Factory.Instruct with instructors = singleton bank; settlers = singleton bank; id = id3; description = "transfer of 300000.0 USD payment"; contextId = None; steps = [third]
 
-  let verifyAccountDisclosureContexts owner ids = Account.verifyAccountDisclosureContexts [(M.fromList $ fmap (\id -> (show id, singleton bank)) ids)] owner
+  let
+    verifyAccountDisclosureContexts owner ids = do
+      a <- (.observers) . snd . head <$> query @Account.T owner
+      a === (M.fromList (fmap (\id -> (show id <> "(0)", singleton bank)) ids))
```

Comment:
You can probably use `$` to get rid of a few brackets

Reply:
for some reason it does like the `$` when combined with the `===`. I changed it to 
```
verifyAccountDisclosureContexts owner ids = do
  o <- (.observers) . snd . head <$> query @Account.T owner
  let o' = M.fromList $ (\id -> (show id <> "(0)", singleton bank)) <$> ids
  o === o'
```
to get rid of some parenthesis.